### ISSUE TITLE
Changes to keyboard.

### ISSUE THREAD

**Issue by loggio** [jurialmunkey/skin.aura#4]
G'day,
Me again...

I made some minor changes to the keyboard to replicate what I had done to my mod of AZ. 
This allows the 'edit control' to be accessed in order to quickly modify any writing instead of having to use the crappy arrow keys on the virtual keyboard.

And I've moved the autocomplete results to show up directly under the edit control instead of at the bottom of the keyboard just like in an internet browser window. Just makes more sense to me.

Anyways, let me know what you think.
Cheers. 

[DialogKeyboard.zip](https://github.com/jurialmunkey/skin.aura/files/1866123/DialogKeyboard.zip)

**Comment by jurialmunkey** [comment]
Ah this is great! Only thing I wasn't a fan of was changing to the underline for the suggestions. I just kept the button style instead (and had to tweak the zoom resize a little to fit it). Thanks!

**Comment by loggio** [comment]
Great, glad you like it.

**Comment by jurialmunkey** [comment]
@loggio - Unfortunately, I had to revert the autocomplete suggestions and put them back to the bottom as I found some usability issues with the remote. However, I've made sure that the edit control is still accessible so that typing can be done directly in the box.

The main issue was that if you type with the remote number keys (e.g. old nokia 3310 SMS style), having the suggestions at the top makes for too much extra navigation. Plus, in daily use I found that the shifting target from the zoom/slide was a little off-putting.
